web_src/css/home.css

```diff
@@ -51,3 +51,12 @@
     float: none;
   }
 }
+
+.ui.message.recently-pushed-branches {
+  align-items: center;
+  padding: 0;
+}
+
+.ui.message.recently-pushed-branches .button {
+  margin: 0;
+}
```

Comment:
I guess there are some helper styles, like `gt-ac`, `gt-p-0` and `gt-m-0`, then we do not need to add more styles.

Reply:
I thought that as well, but wasn't able to find them. Is there an overview with all these helper styles somewhere?

Reply:
`web_src/css/helpers.css`

Reply:
https://github.com/go-gitea/gitea/blob/main/web_src/css/helpers.css

Reply:
Thanks! I've now changed it to use the helper classes instead.